### PR TITLE
Template

### PR DESCRIPTION
**Pull request by tsohleDev**
This a portfolio web design implemented in HTML and CSS

### CODE REVIEW

**Review by Debas-31 — CHANGES_REQUESTED** [review]
> ## Hi @tsohleDev  :wave: 
# Status : INVALID REQUEST :stop_sign: :stop_sign: :stop_sign: 

While you made a great effort in this project, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, 

- [x] The Pull Request shouldn't have a descriptive title telling what changes are inside and a summary describing introduced changes in more detail.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by tsohleDev** [comment]
I  really don't understand, what is it that should be written in a pull request?

**Review by mckent05 — COMMENTED** [review]
Hi  @tsohleDev,

# STATUS: INVALID REQUEST ❌ ❌  

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that doesn't have a descriptive PR title and summary. Your PR summary should include changes made in that PR, this will help anybody viewing the Pr understand easily what the PR is all about.
Please edit your PR summary by clicking on the three dots in the image attached below and selecting the 'edit' button 👍 👍 

pr edit


Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    <link href="https://fonts.googleapis.com/css2?family=Playfair+Display:wght@500&family=Poppins:wght@100&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./style.css">
+</head>
+<body>
+    <header class="toolbar">
+        <p class="nav">Home</p>
+        <p class="nav">Portfoloi</p>
+        <p class="nav">About</p>
+        <p class="nav">Contact</p>
+    </header>
```
Comment (andgarzonmal) [inline index.html]: - [x] Kyndly follow the template for the bobile version,  the mobile version for your template only contains the hamburguer menu in the header. 

### Figma design 
image

### Your design 
image

Comment (andgarzonmal) [inline index.html]: [Optional] As you have already worked on the links thar are displayed on the desktop version is not required to remove what you have done, but for now you will have to set the display as none for those links, I recommendo you doing something similar to this.

image

You will have to set the nav_links class as none first so on your css you an put this 

image

File: style.css [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+  height: 24px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  text-align: right;
+  color: #1a2236;
+}
+
```
Comment (andgarzonmal) [inline style.css:12]: - [x]  Kyndly follow the figma design as much as you can, as you can see in the mobile version of figma there is a background image on the top left corner, plese add it to your project. Take into account that if an image is not part of the content (aesthetic reasons) use CSS background-image. [Check this link](https://stackoverflow.com/questions/492809/when-to-use-img-vs-css-background-image) for more info.

### Figma design
image

### Your project 

image

File: style.css [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+.title {
+  height: 220px;
+  font-family: 'Playfair Display', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 110px;
+  color: #1a2236;
+}
+
+.bio {
```
Comment (andgarzonmal) [inline style.css:45]: - [x] kyndly use the correct fonts and colors, 

### Figma design 
image

### your project 

image

**Review by andgarzonmal — CHANGES_REQUESTED** [review]
# ♻️♻️♻️ STATUS: Changes Required  ♻️♻️♻️


 Hi @tsohleDev ✋🏼 

#### Your project is excellent. You are almost done. However, there are some issues you should fix. Please fix them and your project will be approved.

___ 

### 💯  Highlights 💯
- Your gitHub flow is correct. ✔️
- You set up linter files and no errors. ✔️
- Your pull request is very professional. It has a meaningful title and summary. Keep it up.✔️

### ♻️ Request changes ♻️

- [x] _Check the comments under the review._
___

### Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

### Cheers and Happy coding! 💻 👏
almost


___

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification._

_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._


___

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Comment by tsohleDev** [comment]
Making a mobile website for my portfolio using a figma template

File: style.css [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+  font-weight: 300;
+  font-size: 80px;
+  color: #1a2236;
+  margin: 0;
+}
+
+.bio {
+  width: 60%;
+  height: 84px;
+  font-family: sans-serif;
```
Comment (BertrandConxy) [inline style.css]: - [x] As suggested by the previous reviewer, please use the correct font family in this paragraph and the button below it 😃.
**Your design**
image
**Given design**
image

**HINT** The font family is called `Poppins` sans-serif

**Review by BertrandConxy — CHANGES_REQUESTED** [review]
Hi @tsohleDev, 

Great work on making the changes requested by a previous reviewer 👏 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights 💪 
- No linter errors
- correct flow is used
- good improvement based on the previous reviewer's suggestions
- good PR


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @BertrandConxy in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+  font-style: normal;
+  font-weight: 400;
+  font-size: 14px;
+  line-height: 21px;
+  letter-spacing: -0.01em;
+  color: #1a2236;
+}
```
Comment (Gopxfs) [inline style.css]: As requested by the previous reviewer, kindly use the same font as the one displayed on Figma 😉:
| Figma design  | Your design |
| ------------- | ------------- |
| image  | image  |

**Review by Gopxfs — CHANGES_REQUESTED** [review]
# ♻️ STATUS: REQUIRED CHANGES ♻️   

## Hi @tsohleDev 👋,

You've done well implementing some of the requested changes 👏 , but there are still some which haven't been addressed yet.



## Highlights :rocket:
✔️ No linter errors.
✔️ Good code structure.
✔️ Nice job following some of the previous requests!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions 🗒️  
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @Gopxfs in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by TuyishimireEric — APPROVED** [review]
# Status: APPROVED  :+1: :+1:  
## Hi  @tsohleDev    :hand: 

Thank you for Implementing the requested changes from the previous code reviewer :clap: 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight
- All tests are passing ✔️ 
- Correct use of GitFlow ✔️
- Professional PR ✔️
- Professional commit  ✔️

Image


## Optional suggestions
_NA_

**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**